static/js/TextRange.jsx

```diff
@@ -241,15 +241,15 @@ class TextRange extends Component {
     $(event.target).closest("sup").next("i.footnote").toggle();
     this.placeSegmentNumbers();
   }
-  parashahHeader(data, segment, includeAliyout=false) {
-    // Returns the English/Hebrew title of a Parasha, if `ref` is the beginning of a new parahsah
+  parashahHeader(data, segment, includeAliyot=false) {
```

Comment:
Aliyot is used a lot, this odd spelling was a local thing which I couldn't justify.